Sources/MySQL/Connection/MySQLGeometry.swift

```diff
@@ -0,0 +1,92 @@
+public enum MySQLGeometry: Equatable {
+    case point(x: Double, y: Double)
+
+    fileprivate var wkbType: UInt32 {
+        switch self {
+        case .point: return 1
+        }
+    }
+
+    fileprivate static let headerSize = 9
+}
+
+extension MySQLGeometry: CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case .point(x: let x, y: let y):
+            return "Point(\(x) \(y))"
+        }
+    }
+}
+
+extension MySQLGeometry {
+    func convertToData() -> Data {
+        switch self {
+        case .point(let x, let y):
+            let bufferSize = MySQLGeometry.headerSize + 2 * MemoryLayout<Double>.size
```

Comment:
The MySQL spec requires these to be [8 bytes](https://dev.mysql.com/doc/refman/5.7/en/gis-data-formats.html#gis-wkb-format). I don't know about in Swift, but in C and C++ a double isn't guaranteed to be 8 bytes. Just to have higher precision than a float.

> WKB uses 1-byte unsigned integers, 4-byte unsigned integers, and 8-byte double-precision numbers (IEEE 754 format). A byte is eight bits.

Reply:
In C we'd typically use a static assert to ensure that the double is the size we're expecting or update the algorithm to allow for a double to be another size